exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -135,6 +135,9 @@
   BooleanValidator JSON_EXTENDED_TYPES = new BooleanValidator("store.json.extended_types", false);
   BooleanValidator JSON_WRITER_UGLIFY = new BooleanValidator("store.json.writer.uglify", false);
   BooleanValidator JSON_WRITER_SKIPNULLFIELDS = new BooleanValidator("store.json.writer.skip_null_fields", true);
+  String JSON_READER_SKIP_MALFORMED_RECORDS_FLAG = "store.json.reader.skip_malformed_records";
```

Comment:
Can you change this to 'skip_invalid_records' such that the name is somewhat consistent with the future similar option in DRILL-3764.  In the future the json option would likely be subsumed by the new global option.